### DISCUSSION TITLE
cant get valhalla to take common pedestrian path

### DISCUSSION THREAD

**Discussion by farfromrefug** [valhalla/valhalla#3822] [General]
There is a hiking path here in Grenoble that almost everyone takes especially because other paths are difficult.( same result with graphhoppe, osrm, komoot).
Here are the points:
```js
{lat:45.19593463174191,lon:5.734244249542826}
{lat:45.19895872940955,lon:5.725274942597025}
```
This is the path people use
<img width="448" alt="Screenshot 2022-11-07 at 16 11 18" src="https://user-images.githubusercontent.com/655344/200345479-b7fbd105-73ec-4a22-bc71-63c4073f826a.png">

Now i have been playing with all valhalla pedestrian settings and i cant get valhalla to pick it up.
Either valhalla takes the hardest path (full of stairs) or non direct route while i start right at the bottom of the path (see bottom screenshots). I should mention that i build valhalla tiles with elevation.
it feels like it is a mix of things. A combination of steps_factor/track_factor/service_factor and the rule which define the cost of a edge based on edge "steepness"
On thing that bugs me is that track_factor is actually not manually set but defined through `use_tracks` which means we cant get under 0.8.
The issue could be that it starts with service road which you have to take to start the hike.
If i use a small steps factor it will always use steps even if the steps are very steep.
if i use a high step factor it will go all the way around. This could be explained by the fact that the path i want it to take still has to go through steps.

Does someone has any idea on how to "solve" this? Or is it unsolvable?
<img width="755" alt="Screenshot 2022-11-07 at 16 20 04" src="https://user-images.githubusercontent.com/655344/200346934-da0c40b7-d7a2-4e4e-b3f1-9b35685739de.png">

<img width="587" alt="Screenshot 2022-11-07 at 16 11 26" src="https://user-images.githubusercontent.com/655344/200345749-19a0a541-026a-4e0b-a316-47b56e98238b.png">
<img width="1003" alt="Screenshot 2022-11-07 at 16 11 52" src="https://user-images.githubusercontent.com/655344/200345762-64ed26cf-a3ea-4247-9f08-9758beecc869.png">

**Comment by dnesbitt61** [comment]
This may simply be a matter of the longer distance that the winding path takes? I can move the origin and destination around to take portions of the winding path but many times it prefers a shorter distance alternative to the winding.
